### ISSUE TITLE
Hook up email form

### ISSUE THREAD

**Issue by nappels** [nappels/pltfm-www#12]
Waiting on implementation details from client. I'll update this ticket once we have more details.

**Comment by nappels** [comment]
@everitosan Here are the implementation details:

Going to use a mailchimp form. Here is the markup:

```
<form action="//usemeapp.us7.list-manage.com/subscribe/post?u=5b16e3b63c1bdb0f4a4fa32d8&amp;id=dca37e2e2f" method="post" target="_blank" novalidate>
  <input type="email" value="" name="EMAIL" class="required email" id="mce-EMAIL" placeholder="Email Address">
  <input type="submit" value="Join" />
</form>
```

Feel free to change/add class names or IDs for styling purposes. 

We will also need to create a thank you page for email form submission. I created a new issue for the details on that here: https://github.com/nappels/pltfm-www/issues/15
